src/app/page.tsx

```diff
@@ -1,24 +1,24 @@
 export const fetchCache = 'force-no-store'
 
-import { DndWrapper } from '@/hoc/DndWrapper'
-import { TaskBoard } from './ui/TaskBoard'
-import { z } from 'zod'
-import { WorkflowStateResponse } from '@/types/dto/workflowStates.dto'
+import { createMultipleAttachments, getSignedUrlUpload } from '@/app/actions'
+import { SilentError } from '@/components/templates/SilentError'
 import { apiUrl } from '@/config'
 import { ClientSideStateUpdate } from '@/hoc/ClientSideStateUpdate'
-import { TaskResponse } from '@/types/dto/tasks.dto'
+import { DndWrapper } from '@/hoc/DndWrapper'
+import { RealTime } from '@/hoc/RealTime'
 import { Token, TokenSchema } from '@/types/common'
-import { CopilotAPI } from '@/utils/CopilotAPI'
 import { CreateAttachmentRequest } from '@/types/dto/attachments.dto'
+import { TaskResponse } from '@/types/dto/tasks.dto'
 import { CreateViewSettingsDTO } from '@/types/dto/viewSettings.dto'
-import { createMultipleAttachments, getSignedUrlUpload } from '@/app/actions'
-import { ModalNewTaskForm } from './ui/Modal_NewTaskForm'
-import { RealTime } from '@/hoc/RealTime'
+import { WorkflowStateResponse } from '@/types/dto/workflowStates.dto'
+import { CopilotAPI } from '@/utils/CopilotAPI'
 import { redirectIfTaskCta } from '@/utils/redirect'
+import { UserRole } from '@api/core/types/user'
 import { Suspense } from 'react'
+import { z } from 'zod'
 import { AssigneeFetcher } from './_fetchers/AssigneeFetcher'
-import { SilentError } from '@/components/templates/SilentError'
-import { UserRole } from './api/core/types/user'
+import { ModalNewTaskForm } from './ui/Modal_NewTaskForm'
+import { TaskBoard } from './ui/TaskBoard'
```

Comment:
While you're at it, can you please also change all these relative imports to use aliases imports as well :stuck_out_tongue: